recipes-extended/pam/libpam_1.%.bbappend

```diff
@@ -1 +1,16 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
+
+RCONFLICTS:pam-plugin-faillock += "ni-auth"
+
+pkg_postinst:pam-plugin-faillock:append() {
+	# configure faillock settings
+	sed -i 's/^# audit$/audit/' "${sysconfdir}/security/faillock.conf"
+	sed -i 's/^# silent$/silent/' "${sysconfdir}/security/faillock.conf"
+	sed -i 's/^# deny = 3$/deny = 3/' "${sysconfdir}/security/faillock.conf"
+	sed -i 's/^# fail_interval = 900$/fail_interval = 900/' "${sysconfdir}/security/faillock.conf"
+	sed -i 's/^# unlock_time = 600$/unlock_time = 0/' "${sysconfdir}/security/faillock.conf"
```

Comment:
`faillock.conf` is owned by `libpam-runtime`, which is on the base system image. I structured the change this way so that only users who install `pam-plugin-faillock` will be affected. Can you confirm that it's acceptable for me to modify `faillock.conf` on the base system image?

Reply:
Which package installs the faillock pam `.so`?

Reply:
`/lib/security/pam_faillock.so` is installed by the `pam-plugin-faillock` package.

Reply:
So yes; it's fine that we modify the `faillock.conf` file. We own the distro and can provide whatever defaults we want, including enabling some features by default.

I'm a little confused as to why libpam-runtime owns the `faillock.conf` though. It seems like that should be in `pam-plugin-faillock`.